src/backend/storeManagers/legendary/eos_overlay/eos_overlay.ts

```diff
@@ -14,11 +14,19 @@ import { Path } from '../commands/base'
 import { LegendaryCommand } from '../commands'
 import { sendGameStatusUpdate } from 'backend/utils'
 
-const currentVersionPath = join(legendaryConfigPath, 'overlay_version.json')
-const installedVersionPath = join(legendaryConfigPath, 'overlay_install.json')
-const defaultInstallPath = join(toolsPath, 'eos_overlay')
 const eosOverlayAppName = '98bc04bc842e4906993fd6d6644ffb8d'
 
+function getConstants(): {
+  currentVersionPath: string
+  installedVersionPath: string
+  defaultInstallPath: string
+} {
+  const currentVersionPath = join(legendaryConfigPath, 'overlay_version.json')
+  const installedVersionPath = join(legendaryConfigPath, 'overlay_install.json')
+  const defaultInstallPath = join(toolsPath, 'eos_overlay')
+  return { currentVersionPath, installedVersionPath, defaultInstallPath }
+}
```

Comment:
I think you can simplify this by converting each of the constants into separate functions, I did it like that in this PR since I also had to import `isEnabled` https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/3394/files#diff-efa75d3c5004afcce158bc8224b64a644dd5f88c4a9d0f7a03e1f6f6253963c7R17

Reply:
ill wait for the other one to merge and then ill just rebase it on top of yours